src/common/src/test_utils/rand_array.rs

```diff
@@ -117,6 +117,12 @@ impl RandValue for bool {
     }
 }
 
+impl RandValue for JsonbVal {
+    fn rand_value<R: rand::Rng>(_rand: &mut R) -> Self {
+        JsonbVal::dummy()
```

Comment:
Is this by design? What about adding a `TODO` here?

Reply:
List / Struct do not have this either. Opened #8001 for it.